webpack.config.js

```diff
@@ -160,7 +161,10 @@ if ( calypsoEnv === 'desktop' ) {
 
 const jsLoader = {
 	test: /\.jsx?$/,
-	exclude: /node_modules/,
+	include: [
+		path.resolve( './client' ),
+		fs.realpathSync( './node_modules/notifications-panel' )
```

Comment:
Have we looked into using [peerDependencies](https://nodejs.org/en/blog/npm/peer-dependencies/#the-solution-peer-dependencies)? We can define a much looser range on the notifications project since it'll see less development.

I'm thinking this is probably good for now, but maybe consider doing this for dev only since this is somewhat unusual.